src/Fixer/ControlStructure/NoUnneededBracesFixer.php

```diff
@@ -148,6 +149,12 @@ private function isOverComplete(Tokens $tokens, int $index): bool
         if ($tokens[$index]->isGivenKind(CT::T_GROUP_IMPORT_BRACE_OPEN)) {
             $commaOrCloseBraceIndex = $tokens->getNextTokenOfKind($index, [',', [CT::T_GROUP_IMPORT_BRACE_CLOSE]]);
 
+            $analyzer = new TokensAnalyzer($tokens);
+            if ($analyzer->isBlockMultiline($tokens, $index)) {
+                // Multiline group use: do not remove braces
+                return false;
```

Comment:
```suggestion
            if ($analyzer->isBlockMultiline($tokens, $index)) {
                return false;
```
No need for the comment, it is clear from the code what it is.